lib/server/index.js

```diff
@@ -223,6 +227,11 @@ Server.prototype = {
     var config = this.config;
     var test_page = config.get('test_page');
 
+    // If there is an array of test pages, we cycle through them as the default
+    if (Array.isArray(test_page)) {
+      test_page = test_page[this.testPageCount % test_page.length];
+    }
```

Comment:
Okay, I see now. Good point. I'll attempt method 2, as I expect it will be easier than making sure runner's launch the correct page (since they do not currently navigate directly to a test page).